collect_app/src/main/java/org/odk/collect/android/preferences/PreferenceKeys.java

```diff
@@ -57,6 +57,7 @@
     public static final String KEY_MAP_BASEMAP              = "map_basemap_behavior";
 
     // other keys
+    public static final String KEY_ACTIVITY_LOGGER_EVENT    = "activity_logger_event";
```

Comment:
How about calling this `ACTIVITY_LOGGER_ANALYTICS`? I think that may give a little bit more clarity on what it's for.